codegen/config/config.go

```diff
@@ -138,6 +138,19 @@ func LoadConfig(filename string) (*Config, error) {
 		}
 	}
 
+	for _, filename := range config.SchemaFilename {
+		filename = filepath.ToSlash(filename)
+		var err error
+		var schemaRaw []byte
+		schemaRaw, err = ioutil.ReadFile(filename)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, "unable to open schema: "+err.Error())
```

Comment:
why not return an error like the rest of the error blocks within the function?

Reply:
Just copy paste, thanks :)